tools/bug-report/pkg/content/content.go

```diff
@@ -149,23 +146,29 @@ func GetClusterContext(runner *kubectlcmd.Runner, kubeConfig string) (string, er
 
 // GetNodeInfo returns node information.
 func GetNodeInfo(p *Params) (map[string]string, error) {
-	out, err := p.Runner.RunCmd("describe nodes", "", p.KubeConfig, p.KubeContext, p.DryRun)
-	return retMap("nodes", out, err)
+	out, err := p.Runner.RunCmd("get nodes -o yaml", "", p.KubeConfig, p.KubeContext, p.DryRun)
+	return map[string]string{
+		"nodes": out,
+	}, err
 }
 
-// GetDescribePods returns describe pods for istioNamespace.
-func GetDescribePods(p *Params) (map[string]string, error) {
+// GetPodInfo returns pod details for istioNamespace.
+func GetPodInfo(p *Params) (map[string]string, error) {
```

Comment:
I think GetK8sResources already includes pods. We can add `nodes` there as well?

Confusingly, `all` doesn't mean all. It includes pods, but not nodes.